### ISSUE TITLE
A11y_VSCode_Version Control_Screenreader: Screenreader does not read the error message while creating new file or folder with empty name

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#98929]
[Check out Accessibility Insights!](https://github.com/microsoft/AzureStorageExplorer/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.3.1;#MAS3.3.1;#Win10;#Visual Studio Code;#NVDA;#JAWS; #CAITestPass_May2020;#FTP;#STP;

**Environment Details:**
Application Name: Visual Studio Code
Windows Version: Windows10
Screenreader version:NVDA[2020.1], JAWS 2020.2001.70.400
**Repro Steps:**
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Publish branch to the remote Git repository by navigating to the bottom left pane, arriving at the cloud button next to the git branch and hit ENTER
5.Make changes to the code by refactoring, adding/deleting new lines ; using assistive technologies, observe if SRs will alert user the to blue/green/red indicators suggesting if code has been changed/new line added/line has been removed
6.Compare the changes between the HEAD and current file by navigating to View Bar ---> Git ; Navigate to changes and hit ENTER to activate "Diff View" ; use SR to parse code and observe behavior
7. Stage and commit the changes under View Bar ---> Git  ; Navigate to the "Commit Staged"  button and hit ENTER to make a commit
8. Undo the commit by navigating to the toggle button (same row as commit staged), hit ENTER and use down ARROW to choose "Undo Last Commit" option ; Push a commit following the same process
9. Navigate Git commands under Git output under the "toggle button" in step 7
10.Navigate to New folder or New file button and click on that.
**Actual:**
While navigating through keyboard,Screenreader does not read the error message while creating new file or folder with empty name.
This issue is reproduced with NVDA and Jaws.

**Expected:**
Screenreader should read the error message while creating new file or folder with empty name.

**Attachment link for Reference:**

https://microsoft-my.sharepoint.com/:v:/p/v-nisaho/EVHF_7m6NC5MhlIlK9TLRfkBp291YzQ84EzxYffOQ-kCGw?e=AcFFVI

**Comment by isidorn** [comment]
Please follow these recommendations when filling issues for VS Code:
 
* First try to find duplicates. Before filling any issue be aware of all the [issues tagged with accessibility](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility). It is not a long list, please read through it in detail!
* Always test on VS Code insiders
* Attach gifs inline in the issue. Do not give a link to a 20 MB ppt video
* Do not write User Impact section, it is not useful
* Be responsive, if an issue requires more information please respond in a timely manner
 
Here’s an example of a good concise user filled issue https://github.com/microsoft/vscode/issues/96987
Here’s an example of a not a concise issue with too many details https://github.com/microsoft/vscode/issues/62033
 
The moto should be quality not quantity.

Since this issue does not follow the recommendations I am closing it. Feel free to create a new concise issue.
Thanks

**Comment by nirmalsahoo** [comment]
@isidorn The above two issues are not related to same control.So, we have logged a separate bug.Can you please look into this issue again.
![image](https://user-images.githubusercontent.com/53940787/83502629-8557af80-a4df-11ea-8a8e-1b06b0d683d5.png)

**Comment by isidorn** [comment]
@nirmalsahoo so the issue seems to be "Input box error message not read". Can you please file a new concise issue with minimal steps and we can continue the discussion there. Thanks
